DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -11,10 +11,10 @@
 
 namespace Sonata\CoreBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
```

Comment:
I mean that all bundles have an implicit requirement on FrameworkBundle.
Take this line: `$container->getDefinition('form.extension')`
Where does this `form.extension` come from ? FrameworkBundle.
Thus, this PR adds no additional requirements at all.